variables.tf

```diff
@@ -0,0 +1,47 @@
+variable "location" {
+  description = "Azure location to deploy the resource."
+  type        = string
+  default     = "UK South"
+}
+variable "env" {
+  description = "Name of the environment to deploy the resource."
+  type        = string
+}
+variable "product" {
+  description = "Name of the product."
+  type        = string
+}
+variable "builtFrom" {
+  description = "Name of the GitHub repository this application is being built from."
+  type        = string
+}
+variable "project" {
+  default     = "acme"
+  description = "Name of the project."
+}
+variable "aspskusize" {
+  default     = "Y1"
+  description = "SKU size for App Service Plan."
+}
+variable "aspskutier" {
+  default     = "Dynamic"
+  description = "SKU tier for App Service Plan."
+}
+
+variable "dns_zone_subscription_id" {
```

Comment:
this should probably be done by passing an additional provider and then using https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/client_config

Reply:
done